routes/recache.py

```diff
@@ -36,7 +36,7 @@ def log_error(url, status):
     log.close()
 
 
-def get_all_huc_jsons():
+def get_all_jsons():
     """*** Unused for now ***
        For all HUC types defined in the LUTs, load the JSON files into a
```

Comment:
Update the comment to reflect the new functional goal here.

Reply:
Updated the comment! Thanks for catching this!